scripts/west_commands/runners/nrf_common.py

```diff
@@ -34,6 +35,54 @@
     }
 }
 
+class SysbuildConfiguration(BuildConfiguration):
```

Comment:
Please move this to `runners.core` and, if possible, derive instead from a common base class so that you can reuse all the code

Reply:
`SysbuildConfiguration` moved into `runners.core`, as requested.